py/picca/fitter2/pk.py

```diff
@@ -1,17 +1,25 @@
 import scipy as sp
+import numpy as np
```

Comment:
are you sure you need numpy? Can't you use scipy?

Reply:
whatever numpy is included in scipy I will change

Reply:
sp.absolute()